dotCMS/src/integration-test/java/com/dotcms/publishing/job/SiteSearchJobImplTest.java

```diff
@@ -448,74 +454,220 @@ public void Test_Incremental_Job_Test_Pages_Are_Found_Create_And_Publish_New_Pag
 
     }
 
-    /**
-     * Given sceneario: Multi-language content referenced from a page. Create a Site-Search run including all the languages of the content and
-     * check the two versions made it into the resulting index.
-     */
+    static class TestCaseSiteSearch {
+        boolean defaultPageToDefaultLanguage;
+        boolean defaultContentToDefaultLanguage;
+        boolean siteSearchDefaultLanguage;
+        boolean siteSearchSecondLanguage;
+        boolean siteSearchThirdLanguage;
+        boolean createContentInDefaultLanguage;
+        boolean createContentInSecondLanguage;
+        boolean createContentInThirdLanguage;
+        boolean createPageInDefaultLanguage;
+        boolean createPageInSecondLanguage;
+        boolean expectedResultsWhenSearchingContentInDefaultLanguage;
+        boolean expectedResultsWhenSearchingContentInSecondLanguage;
+        boolean expectedResultsWhenSearchingContentInThirdLanguage;
+
+        public TestCaseSiteSearch(boolean defaultPageToDefaultLanguage,
+                boolean defaultContentToDefaultLanguage, boolean siteSearchDefaultLanguage,
+                boolean siteSearchSecondLanguage, boolean siteSearchThirdLanguage,
+                boolean createContentInDefaultLanguage,
+                boolean createContentInSecondLanguage, boolean createContentInThirdLanguage,
+                boolean createPageInDefaultLanguage,
+                boolean createPageInSecondLanguage,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'createPageInSecondLanguage' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5303548)